### DISCUSSION TITLE
Media player page

### DISCUSSION THREAD

**Discussion by cr4shydlo** [Blackymas/NSPanel_HA_Blueprint#1860] [Ideas]
Control Functions: Users will have the ability to control media playback functions such as play, pause, stop, and skip directly from the NSPanel.

Radio Station Selection: The Media Player Card will allow users to browse and select from available radio stations conveniently within the NSPanel interface.

Speaker Management: Users will be able to manage speaker output settings, including volume control and speaker selection, directly from the NSPanel.

**Comment by edwardtfn** [comment]
- #1859
